code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useParams, useHistory, Link } from "react-router-dom";
+
+const MovieDetails = ({ allMovies }) => {
+  const { movieId } = useParams();
+  const movidIdNum = parseInt(movieId);
+  const history = useHistory();
+
+  const matchingMovie = allMovies.find((movie) => movie.id === movidIdNum);
+
+  if (!matchingMovie) {
+    history.push("/");
+    return <span>404</span>;
+  }
+
+  const backgroundStyles = {
+    backgroundImage: `url(https://image.tmdb.org/t/p/w1280${matchingMovie.backdrop_path})`,
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    backgroundSize: "cover",
+    backgroundRepeat: "no-repeat",
+    minHeight: "100vh",
```

Comment:
I looked at it in mobile view and can see that when you shrink the size of the screen you wont be able to see much of the background image. If you would set background-position: bottom; it would be centered in both desktop/tabled and mobile view! :) 
![codereview sarah and ru](https://user-images.githubusercontent.com/81567468/141450146-749b2f72-edd6-4aa8-bec6-698809c1080b.PNG)
![codereview sarah and ru2](https://user-images.githubusercontent.com/81567468/141450154-c1b895a8-ee8f-4f75-ac56-9d5fcae9a896.PNG)